models/dynamic-starwars/src/model.rs

```diff
@@ -227,7 +229,7 @@ pub fn schema() -> Result<Schema, SchemaError> {
                     let starwars = ctx.data::<StarWars>()?;
                     let humans = starwars.humans();
                     Ok(Some(FieldValue::list(
-                        humans.into_iter().map(FieldValue::borrowed_any),
+                        humans.into_iter().map(|v| FieldValue::borrowed_any(v)),
```

Comment:
Maybe we should change `borrowed_any(obj: &'a (impl Any + Send + Sync))` to `borrowed_any(obj: &'a (impl Any + Send + Sync + ?Send))` to avoid this unnecessary `map` calls?